src/modules/collection/reducer.ts

```diff
@@ -227,7 +227,8 @@ export function collectionReducer(state: CollectionState = INITIAL_STATE, action
             ...state.data[collection.id],
             lock: collection.lock
           }
-        }
+        },
+        loading: loadingReducer(state.loading, action)
```

Comment:
this line is to update the loading state when publishing a collection is successful